packages/rpc/src/index.ts

```diff
@@ -13,11 +13,9 @@
 // limitations under the License.
 //
 
-import { Class, Doc, DocumentQuery, Ref, Storage, Tx } from '@anticrm/core'
 import { component, Component, PlatformError, Severity, Status, StatusCode } from '@anticrm/status'
 
-export type ReqId = string | number
-
+export type ReqId = string
```

Comment:
Why `ReqId` is not a `string | number` anymore? -- this is according to JsonRPC spec.

Reply:
Sure, let me make it according to spec.